tw-tasks-executor/gradle.properties

```diff
@@ -1,2 +1,2 @@
-version=1.3.1
+version=1.4.0
```

Comment:
This could be a major bump / breaking change in order to force people to think about their alternative adoption of alerting manager, but might be abit extreme.